README.md

```diff
@@ -24,15 +43,16 @@ With Docker:
 
 
 ```sh
-docker run -u [USER] -v [LOCAL_VOLUME_MAPPING]:/restore -e BACKUP_PROVIDER_TOKEN=[BACKUP_PROVIDER_TOKEN] \
-  opinari/freighter restore --remoteFilePath [REMOTE_FILE_PATH] --restoreFilePath /restore/[LOCAL_FILE_PATH] 
+docker service create -u [USER] -v [LOCAL_VOLUME_MAPPING]:/restore \
+  --secret source=.freighter.yml,target=/tmp/.freighter.yml \
```

Comment:
Doesnt the use of "--secret" always assume this has to be run within an active Docker Swarm to work?